cmd/orb-api/config.example.yml

```diff
@@ -0,0 +1,26 @@
+listener:
+  bind_addr: "0.0.0.0:8080"
+  tls: 
```

Comment:
If this section is empty the api will start in plain text. If auto is specified it will try to use autocert which requires a public ip. If you specify a cert and key path it will prefer those over anything else.